### PR TITLE
Feature/versa tvl

### PR DESCRIPTION
**Pull request by denis-post**
##### Twitter Link:
https://twitter.com/VersaGamesHQ

##### List of audit links if any:
https://paladinsec.co/projects/versagames/

##### Website Link:
https://versagames.io/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://versagames.io/assets/tokens/versa.svg

##### Current TVL:
image

##### Chain:
Cronos

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
versagames

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
N/A

##### Short Description (to be shown on DefiLlama):
NFT Gaming Platform powered by $VERSA token. Join the future of Web 3.0 Gaming

##### Token address and ticker if any:
VERSA: 0x00D7699b71290094CcB1a5884cD835bD65a78c17

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Gaming

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
N/A

##### forkedFrom (Does your project originate from another project):
N/A

##### methodology (what is being counted as tvl, how is tvl being calculated):
TVL is calculated as value of tokens in VERSA-xVERSA staking

### CODE REVIEW

File: projects/VersaGames/config.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
+const config = {
+  treasuryAddress: "0x5505ad96d4B1181c00248C8ac2a9A865065FD69D",
```
Comment (g1nt0ki) [inline projects/VersaGames/config.js:2]: this treasury is a private wallet, not a contract?

File: projects/VersaGames/config.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  opsAddress: "0x3dABd1Ac5E89e01e928fca232672c4848C6E3097",
```
Comment (g1nt0ki) [inline projects/VersaGames/config.js:3]: looks like this is also EOA, sorry we cant count this towards tvl or treasury

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

File: projects/VersaGames/index.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+    versaVvsLP.farmPoolId,
+    chain,
+    chainBlocks[chain.name]
+  );
+
+  sumMultiBalances(balances, otherBalances.concat(farmPendingTokensBalances));
+  return balances;
+}
+
+async function tvl(timestamp, ethBlock, chainBlocks) {
```
[the remark this answers is not in the document]
Comment (denis-post) [reply projects/VersaGames/index.js:200]: got it

File: projects/VersaGames/config.js (hunk 1 above)
Comment (denis-post) [reply projects/VersaGames/config.js:2]: yes, treasury and operations & funds wallets are private wallets owned by VersaGames

**Comment by g1nt0ki** [comment]
@denis-post can you also explain the pool2 part? I dont see any LP staking option on your site

File: projects/VersaGames/config.js (hunk 1 above)
Comment (denis-post) [reply projects/VersaGames/config.js:2]: fixed, removed private wallets from tvl

File: projects/VersaGames/config.js (hunk 2 above)
Comment (denis-post) [reply projects/VersaGames/config.js:3]: removed it

File: projects/VersaGames/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (denis-post) [reply projects/VersaGames/index.js]: fixed it, set tvl key to 0

**Comment by denis-post** [comment]
@g1nt0ki I think pool2 falls under the same EOA scenario. Treasury private wallet holds all tokens VERSA-USDC LP and half of tokens from  VERSA-VVS. I will remove it

**Comment by denis-post** [comment]
@g1nt0ki I updated everything as per your feedback, please review

**Comment by llamatester** [comment]
The adapter at projects/VersaGames exports TVL: 
        
 
 
    cronos                    0
    cronos-staking            1.48 M
    staking                   1.48 M
    
    total                    0

**Pull request merged**